File: Scientificomputing/homework/homework/1/homework/homework/count/java/Counting.java
import java.util.*;
import java.io.*;
class Counting
{
    
     Scanner in = new Scanner(System.in);
     int n = in.nextInt();
      
      TimeContainer container=new TimeContainer();
      Timing timer1=new Timing(container, "timer1");

      private float[] count = new float[n];
      public Counting() {
      int i,j,steps;
      steps=20;
      

      for (j=0; j<=steps; j++) {
          timer1.start();
      for (i=0; i < n; i++) {
           count[i] = i+1;
           System.out.println("count[" + i + "]= " + count[i]);
         }
         timer1.stop();
      }

      container.dumpTimings(n);
    }

	// accessor

    public static void main(String[] args) {
         System.out.println("Please Input a Integer:");
        Counting counting = new Counting();
    }
   

}

File: Scientificomputing/homework/homework/1/homework/homework/count/c++/Counting.cpp
#include <stdio.h>
#include <stdlib.h>
#include "timing.h"

using namespace std;
    
int main()
{
    //ofstream myfile1;
    //myfile1.open ("home1.txt");
    
	Timings ts;
	Timer t1(ts, "newtim");
 	        
    int n,steps;
    steps=20;
    
    cout << "Enter the value of n (Press 0 to quit): " ;
    cin >> n;
    
    cout<< "The value entered is : " << n<<endl;
    cout<< "Counting from 1 thru "<< n << endl;

    float count[n+1];

    for (int j=1; j<=steps; j++)
      {
       t1.start(Timer::SELF);
        for (int i=0; i < n; i++)       
          {
            count[i] = i+1;
            cout<< "count[" << i << "] = " << count[i] << endl;
           } 
       t1.end();       
       }
     ts.dumpTimings(steps,n);
     return 0;             
}

